src/mongo/db/auth/role_graph_builtin_roles.cpp

```diff
@@ -430,6 +430,8 @@ void addBackupPrivileges(PrivilegeVector* privileges) {
         privileges, Privilege(ResourcePattern::forAnyResource(), ActionType::listCollections));
     Privilege::addPrivilegeToPrivilegeVector(
         privileges, Privilege(ResourcePattern::forAnyResource(), ActionType::listIndexes));
+    Privilege::addPrivilegeToPrivilegeVector(
+        privileges, Privilege(ResourcePattern::forAnyNormalResource(), ActionType::startBackup));
```

Comment:
Can you explain what is going here? You add startBackup action to the list of what?